spacemacs/funcs.el

```diff
@@ -143,6 +143,18 @@ the current state and point position."
       (sp-newline)
       (setq counter (1- counter)))))
 
+;; eval lisp helpers
+(defun spacemacs/eval-last-sexp ()
+  (interactive "P")
+  (spacemacs/message "Executing code ...")
+  (eval-last-sexp))
+
+(defun spacemacs/eval-region ()
+  (interactive)
+  (spacemacs/message "Executing code ...")
+  (eval-region (region-beginning) (region-end))
+  (evil-normal-state))
+  
```

Comment:
I am not favorable for this wrappers just to display a text in the minibuffer, it will be a burden to make it consistent with the other languages. Moreover there is now `eval-sexp-fu` shipped with spacemacs which gives a visual feedback when something is evaluated.